traefik/templates/requirements.yaml

```diff
@@ -35,3 +35,7 @@
 {{- if and (semverCompare "<3.3.0-0" $version) (.Values.experimental.abortOnPluginFailure)}}
   {{- fail "ERROR: abortOnPluginFailure is an experimental feature only available for traefik >= v3.3.0." }}
 {{- end }}
+
+{{- if and (semverCompare "<3.2.0-0" $version) (.Values.providers.kubernetesGateway.nativeLBByDefault)}}
```

Comment:
This has been introduced in v3.0.

cf. https://github.com/traefik/traefik/pull/10323